odl_video/settings.py

```diff
@@ -488,7 +497,7 @@
     },
     'redis': {
         "BACKEND": "django_redis.cache.RedisCache",
-        "LOCATION": CELERY_BROKER_URL,
+        "LOCATION": REDIS_URL,
```

Comment:
`CELERY_BROKER_URL` will default to `REDIS_URL` if the env. variable isn't set, so maybe it might be best to not change this line, @noisecapella do you want to weigh in?  That seems to be the pattern used in open-discussions & micromasters as well:
```
CELERY_BROKER_URL = get_string("CELERY_BROKER_URL", REDIS_URL)
```

Reply:
It looks like Giovanni added that line in a commit to add mandatory settings, I don't think `REDIS_URL` has any special meaning to Celery or django-redis. I think the change here makes sense to keep the celery redis broker separate from the redis cache Django